File: MJ/1.17/IO/2442.py
N = int(input())

    
for i in range(1, N+1):
    for _ in range(N-i):
        print(' ', end = '')
    for _ in range(i-1):
        print('*', end = '')
    print('*', end = '')
    for _ in range(i-1):
        print('*', end = '')
    print()

File: Lami/01.31/2442.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
let input = fs.readFileSync(filePath).toString().split('\n');

function solution() {
  const line = Number(input[0]);
  for (let i = 1; i <= 2 * line - 1; i += 2) {
    console.log(' '.repeat((2 * line - 1 - i) / 2) + '*'.repeat(i));
  }
}

solution();
